map.js

```diff
@@ -436,8 +379,10 @@ async function initImgs(){
 }
 
 function onMouseClick(mouseLoc){
-    console.log("click");
-    let handled = topbar.click(mouseLoc);
+    if(window.debug){
+        console.log("click at screen: " + mouseLoc+", map: "+screenSpaceToMapSpace(mouseLoc));
+    }
+    let handled = map_topbar.click(mouseLoc);
```

Comment:
each object has a chance to intercept the mouse click, and if it does, then it stops propagation.